### DISCUSSION TITLE
How to show graphs from fluent

### DISCUSSION THREAD

**Discussion by jharming** [ansys/pyfluent#1080] [Q&A] [answered]
I tried using the mixing elbow as an example to pyfluent.

I tried using spyder and jupyter notebook.

I can run the example in both spyder and jyputer notebook but I cant see any graphs. 
`solver.tui.display.objects.create(
    "contour",
    "contour-temp",
    "filled?",
    "yes",
    "node-values?",
    "yes",
    "field",
    "temperature",
    "surfaces-list",
    "symmetry-xyplane",
    "()",
    "coloring",
    "smooth",
    "quit",
)`

When I come to the command above nothing seems to happen. I was expecting a png file in the python folder I run from and a picture.

**Comment by jharming** [comment, marked as answer]
I found out tha tI could use the graphics module see [https://fluentvisualization.docs.pyansys.com/examples/00-postprocessing/post_processing_exhaust_manifold.html#sphx-glr-examples-00-postprocessing-post-processing-exhaust-manifold-py](url)
Here is the example for mesh.

graphics = Graphics(session=solver_session)
mesh1 = graphics.Meshes["mesh-1"]
mesh1.show_edges = True
a = mesh1.surfaces_list.allowed_values
mesh1.surfaces_list = a
mesh1.display("window-1")

**Reply by amscosta** [reply]
Hello, 
How can I save what is displayed in the window-1?

For instance in the mixing_elbow.py, there are these following lines :

solver.tui.display.objects.create(
    "contour",
    "contour-vel",
    "filled?",
    "yes",
    "node-values?",
    "yes",
    "field",
    "velocity-magnitude",
    "surfaces-list",
    "symmetry-xyplane",
    "()",
    "coloring",
    "banded",
    "quit",
)

When I execute(after the previous line of the script) nothing is shown.

One more thing : What it is the usage of the following comments in that script :

###############################################################################
# .. image:: /_static/mixing_elbow_014.png
#   :width: 500pt
#   :align: center

**Comment by MingjieZhangNJU** [comment]
I am using pyfluent, ansys 22.2 student version, visual studio code. I tried to run the three examples for pyfluent. Everything works fine; but the graphic doesn't show. I CAN see somethings flashes for one second. It may be the graphic but it has difficult to show up correctly.
